src/System.Data.SqlClient/tests/FunctionalTests/SqlConnectionTest.cs

```diff
@@ -30,6 +30,29 @@ public void ConnectionTest()
             Assert.Null(e);
         }
 
+
+        [Fact]
+        [PlatformSpecific(TestPlatforms.Windows)]  // Integ auth on Test server is supported on Windows right now
+        public void IntegratedAuthConnectionTest()
+        {
+            Exception e = null;
+
+            using (TestTdsServer server = TestTdsServer.StartTestServer())
+            {
+                try
+                {
+                    SqlConnectionStringBuilder builder = new SqlConnectionStringBuilder(server.ConnectionString);
+                    builder.IntegratedSecurity = true;
+                    SqlConnection connection = new SqlConnection(builder.ConnectionString);
+                    connection.Open();
+                }
+                catch (Exception ce)
+                {
+                    e = ce;
+                }
+            }
+            Assert.Null(e);
```

Comment:
Personally I think it's fine to have a test without asserts: it's implicitly verifying that everything runs to completion, and if there's something you can assert as well, bonus.  But if you do end up keeping something like what you have, at least use a better assert: as it is, if it does fail, you're going to lose all details on the exception, other than its type name.